src/regions/ca/info/common/prevent-infection.md

```diff
@@ -97,7 +97,17 @@ Refer to the [guidance on cleaning and disinfecting public spaces](https://www.c
 
 Medical masks, including surgical, medical procedure face masks and respirators (like N95 masks), must be kept for health care workers and others providing direct care to COVID-19 patients.
 
-Wearing a non-medical mask or face covering while out in public is optional. If you do choose to wear one, refer to our [guidelines on wearing non-medical masks and how to make your own](https://www.canada.ca/en/public-health/services/diseases/2019-novel-coronavirus-infection/prevention-risks/instructions-sew-no-sew-cloth-face-covering.html).
+Wearing a non-medical mask or face covering while out in public is recommended for periods of time when it is not possible to consistently maintain a 2-metre physical distance from others, particularly in crowded public settings, such as:
+
+- stores
```

Comment:
```suggestion
- Stores
```

Reply:
Can we please capitalize bullet points? :) All other lists on the site have them capitalized, and it's nice to have everything uniform

Reply:
Good point !! Je ferai attention à ça maintenant ;)